samples/v1/types_config.go

```diff
@@ -64,6 +64,15 @@ type ConfigSpec struct {
 	// content but the operator will not recreate(or update) anything
 	// listed here.
 	SkippedTemplates []string `json:"skippedTemplates,omitempty" protobuf:"bytes,6,opt,name=skippedTemplates"`
+
+	// skippedhelmcharts specifies names of helm charts that should NOT be
+	// created/updated.  Admins can use this to allow them to delete content
+	// they don’t want.  They will still have to manually delete the
+	// content but the operator will not recreate(or update) anything
+	// listed here.
```

Comment:
This may want to be reworded to be more direct. It's currently a little confusing and if I were to stop reading half way through, I may not realise that I as the user, have to delete the content myself.

Reply:
updated it